sympy/solvers/diophantine.py

```diff
@@ -55,12 +55,12 @@ def diophantine(eq, param=symbols("t", integer=True)):
     >>> from sympy.solvers.diophantine import diophantine
     >>> from sympy.abc import x, y, z
     >>> diophantine(x**2 - y**2)
-    set([(-t_0, -t_0), (t_0, -t_0)])
+    set([(t_0, -t_0), (t_0, t_0)])
```

Comment:
check on how the elements are sorted, too, because line 3950 looks like 1st and 3rd are reversed